src/Plugin.php

```diff
@@ -261,23 +261,26 @@ private function cleanInstalledPaths()
     private function updateInstalledPaths()
     {
         $changes = false;
-        $codingStandardPackages = $this->getPHPCodingStandardPackages();
 
+        $searchPaths = [getcwd()];
```

Comment:
For completeness sake: although usually discouraged using `getcwd` (and consorts) is the "Composer Way" of doing things.